README.rst

```diff
@@ -91,14 +91,12 @@ In submission mode you can view the self text for a submission and browse commen
 Configuration
 -------------
 
-RTV will read a configuration file located at **~/.rtv**.
+RTV will read a configuration file located at ``$XDG_CONFIG_HOME/rtv/rtv.cfg``.
```

Comment:
Good point... added mention of default location if `XDG_CONFIG_HOME` is not set.